rfcs/_process.md

```diff
@@ -0,0 +1,144 @@
+# test262 RFC process
+
+Many changes, including bug fixes and new tests, can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the test262 maintainers, community of test writers, and consumers of the test suite.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the test suite and interpreter instructions, so that all stakeholders can be confident about the direction the test suite is evolving in.
+
+## Table of contents
+
+- [Opening](#test262-rfc-process)
+- [Table of contents](#table-of-contents)
+- [When you need to follow this process](#when-you-need-to-follow-this-process)
+- [Before creating an RFC](#before-creating-an-rfc)
+- [What the process is](#what-the-process-is)
+- [The RFC life-cycle](#the-rfc-life-cycle)
+- [Reviewing RFCs](#reviewing-rfcs)
+- [Implementing an RFC](#implementing-an-rfc)
+- [RFC postponements](#rfc-postponements)
+- [Acknowledgements](#acknowledgements)
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial" changes to the test262 suite and harness.
+What constitutes a "substantial" change is evolving based on community norms, but may include the following.
+
+- Additions or changes to [INTERPRETING.md](https://github.com/tc39/test262/blob/main/INTERPRETING.md) that require consumers of the test suite to make changes to their test runners.
+- Additions to [features.txt](https://github.com/tc39/test262/blob/main/features.txt) that are not new Stage 3 proposals, or removals of feature flags.
+- Substantial changes and new features in the [harness files](https://github.com/tc39/test262/tree/main/harness).
+
+Some changes do not require an RFC:
+
+- Adding test coverage.
+- Bug fixes.
+- Adding features to [features.txt](https://github.com/tc39/test262/blob/main/features.txt) that are new Stage 3 proposals.
+- Bug fixes to the [harness files](https://github.com/tc39/test262/tree/main/harness).
+
+If you submit a pull request for a "substantial" change without going through the RFC process, it may be closed with a polite request to submit an RFC first.
+
+## Before creating an RFC
+
+Laying some groundwork ahead of the RFC can make the process smoother.
+Getting people to agree with an RFC often requires some effort towards consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking the idea over on our [Matrix channel](https://matrix.to/#/#test262-maintainers:matrix.org), discussing the topic on our [discussion forum](https://es.discourse.group/c/test262/FIXME), and occasionally posting "pre-RFCs" on the forum.
```

Comment:
We should create this tag in the Discourse.

Reply:
During the meeting we discussed that it's probably not necessary to have a tag at this time — communication via the GitHub repo and Matrix channels seems to be sufficient right now